sdk/personalizer/Azure.AI.Personalizer/src/GlobalSuppressions.cs

```diff
@@ -4,3 +4,5 @@
 using System.Diagnostics.CodeAnalysis;
 
 [assembly: SuppressMessage("Usage", "AZC0016:Invalid ServiceVersion member name.", Justification = "Generated code: https://github.com/Azure/autorest.csharp/issues/1524", Scope = "type", Target = "~T:Azure.AI.Personalizer.PersonalizerClientOptions.ServiceVersion")]
+[assembly: SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "<Pending>", Scope = "member", Target = "~P:Azure.AI.Personalizer.DecisionContext.SharedFromUrl")]
```

Comment:
Can you point me to the lines of code that require suppression? Warnings are generally there for a reason.

Reply:
I have removed this line as I don't need it. You will see it in my next commit.